releng/org.eclipse.xtext.sdk.feature/feature.xml

```diff
@@ -67,10 +67,6 @@ SPDX-License-Identifier: EPL-2.0
          version="0.0.0"/>
 
    <requires>
-      <import plugin="org.eclipse.emf.ecore" version="2.20.0" match="greaterOrEqual"/>
```

Comment:
I'd argue that the xtext.sdk requires the emf.sdk instead of a "random" list of emf plugins. Is it possible to edit *.ecore files reasonably without the UI features of EMF?

Reply:
I agree it's better to require the full EMF SDK than single plugins, which, anyway, would be installed automatically due to required bundles of several xtext plugins.

Reply:
we had emf sdk before and it was removed ...

Reply:
IIRC, it was removed because of the Tycho bug that has been fixed now, wasn't it?

Reply:
yes. so we dont need this removal, but i wonder why we need it at all

Reply:
If there are Xtext bundles that already require emf.ecore with at least version 2.20.0 or greater (and I guess there are some), then that import is useless